src/teleop/jetson/jetson_teleop.py

```diff
@@ -83,6 +85,10 @@ def teleop_drive_callback(self, msg: Joy):
 
 
 class ArmControl:
+
+    _arm_mode = "arm_disabled"
+    _arm_mode_lock = Lock()
```

Comment:
Wait sorry I hadn't realized this before but are these supposed to be class variables (kinda like statics?) Seems to me like these should just be member variables defined in the constructor with self._arm_mode = "arm_disabled" and self._arm_mode_lock = Lock().

You can still label the type at the top of the class like this by just using a colon and the type if you don't use an equals sign

Reply:
I thought I had some kind of rationale for doing this but it's not coming to me right now, so i'll just swap it.